lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.js

```diff
@@ -311,16 +321,18 @@ export default function BasicLTILaunchApp() {
 
   const content = (
     <div
-      className={classnames('BasicLTILaunchApp__content', {
-        'is-hidden': !contentReady || errorState,
+      className={classnames('flex flex-col h-full', {
+        invisible: !showContent,
+        visible: showContent,
       })}
+      data-testid="content-wrapper"
     >
       {iFrameWrapper}
     </div>
   );
 
   return (
-    <div className="BasicLTILaunchApp">
+    <div className="h-full">
```

Comment:
Would `grow` be ever needed here?

Reply:
I don't believe so: this isn't in a flex layout here.